check-cabal-project.sh

```diff
@@ -0,0 +1 @@
+/nix/store/19m3ybns2whd3lnc4jydzd1zhw93xq0r-check-cabal-project
```

Comment:
Shouldn't be committed. Perhaps we should add it to .gitignore though.

Reply:
done